libs/ui/src/diagnostics/uninterruptible_power_supply_section.tsx

```diff
@@ -0,0 +1,44 @@
+import { assert } from '@votingworks/basics';
+import { DiagnosticRecord } from '@votingworks/types';
+import { H2, P } from '../typography';
+import { InfoIcon, SuccessIcon, WarningIcon } from './icons';
+
+export interface UpsSectionProps {
+  upsSectionContents?: JSX.Element;
```

Comment:
What is this prop used for? I might be missing something about how the diagnostics pages are wired up

Reply:
`*Section` diagnostic components are used in 2 places, the backend rendering of the PDF and the frontend rendering of the UI components. It's useful for each place to pass children (or not). In the VxCentralScan PR I pass a button to toggle the modal from the frontend, but not the backend PDF renderer.

In practice, it feels like the diagnostic pages are pretty disjointed across all the apps, so this pattern seems most used in VxCentralScan and least used in VxMarkScan.